<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>518</width>
        <height>454</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Directory:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="directoryAddressEdit" >
            <property name="inputMask" >
             <string/>
            </property>
            <property name="text" >
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="directoryPortEdit" >
            <property name="inputMask" >
             <string/>
            </property>
            <property name="text" >
             <string>12345</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>User alias:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="directoryAliasEdit" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Connection:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Local port:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="localPortEdit" >
            <property name="inputMask" >
             <string/>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Preffered RTP port:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="prefferedRTPPortEdit" >
            <property name="inputMask" >
             <string/>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>518</width>
        <height>454</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Audio</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Preffered incoming audio codecs</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QListWidget" name="incomingCodecsList" />
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" >
            <item>
             <spacer name="verticalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="incomingUpButton" >
              <property name="text" >
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="incomingDownButton" >
              <property name="text" >
               <string>Down</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Preffered outgoing audio codecs</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QListWidget" name="outgoingCodecsList" />
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" >
            <item>
             <spacer name="verticalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="outgoingUpButton" >
              <property name="text" >
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="outgoingDownButton" >
              <property name="text" >
               <string>Down</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>518</width>
        <height>454</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Conferences</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8" >
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Mixer</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Mixer ICE port:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="lineEdit" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Mixer RTP port:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lineEdit_2" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Mixer alias:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="lineEdit_3" />
          </item>
          <item row="3" column="0" >
           <spacer name="verticalSpacer_4" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>tabWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>mixerRTPPortEdit</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>182</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>250</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
